### ISSUE TITLE
Remove scrollbar from Bible view

### ISSUE THREAD

**Issue by timbze** [AndBible/and-bible#3344]
Request from https://support.andbible.org/scp/tickets.php?id=2775.

If it's removed it should only be for "infinite scroll" content I think.

Is #2397 the same request?

**Comment by tuomas2** [comment]
It is related but not same. Combining with that, we would like to have setting to have it either always on, or never displayed, in addition to the default setting.

**Comment by Noitarud** [comment]
It is like a warning for how close it is to loading more content, less relavent with much loaded (short bar).
AB places a big white space at the end of a book and the bar indicates scrolling is working to get away from the end.
It maybe too conspicuously coloured for some people, intermittent gentle scrolling causes flash - maintain its appearence while still touching screen?

**Comment by meichthys** [comment]
I agree that the scroll bars are hardly meaningful in the infinite scroll mode.
It does seem like the best default would be to hide scrollbars in infinite scroll mode and show them otherwise.

**Comment by tuomas2** [comment]
Estimate 30 minutes. To implement or not to implement is more about deciding what is good for the general audience. I have not been bothered by small scrollbars, but I could try also removing them, perhaps I won't miss them.

**Comment by meichthys** [comment]
If we were to keep the scroll bars, it would make sense to make the scroll bar sync with chapter divisions. So when scrolling past the end of a chapter, the scroll bar moves back to the top.
